### DISCUSSION TITLE
Asset create api not working

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Knowlg/Community#127] [General Discussions]
I am trying to create a certificate by using the following API but it is not working
giving a response as Bad request

`curl --location 'https://dev.sunbirded.org/api/asset/v1/create' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer <Authorization>' \
--header 'x-authenticated-user-token: <x-authenticated-user-token>' \
--header 'X-Channel-ID: <channelID>' \
--data '{
    "request": {
        "asset": {
            "name": "Single",
            "code": "Single",
            "contentType": "Asset",
            "createdBy": "155ce3c5-713e-4749-bc1c-95d09c640914",
            "mimeType": "image/svg+xml",
            "license": "CC BY 4.0",
            "primaryCategory": "Certificate Template",
            "mediaType": "image",
            "certType": "cert template layout",
            "channel": "<channelID>",
            "language": [
                "English"
            ]
        }
    }
}'`

cc: @vinukumar-vs , @Krishnaj20 , @ pallakartheekreddy

**Comment by pallakartheekreddy** [comment]
@rajeevsathish above info not sufficient to help. Can you share the content service log while calling above API to help you.

**Reply by pallakartheekreddy** [reply]
@rajeevsathish are you still able facing this? If yes, Please share the more logs to help you. If not, please provide the details how did you fixed the issue. It will be helpful for others.

**Reply by pallakartheekreddy** [reply]
@rajeevsathish Hope your query is now resolved. Please mark the thread as answered

**Reply by pallakartheekreddy** [reply]
@rajeevsathish As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
